corehq/apps/export/esaccessors.py

```diff
@@ -26,6 +27,12 @@ def get_case_export_base_query(domain, case_type):
             .sort("opened_on"))
 
 
+def get_sms_export_base_query(domain):
+    return (SMSES()
+            .domain(domain)
+            .sort("date"))
```

Comment:
`processed` isn't included in the sms mapping so this will require a reindex https://github.com/dimagi/commcare-hq/blob/8667a4159ad39d3307f92a3812d74006d610fd1b/corehq/pillows/mappings/sms_mapping.py#L36